src/kubernetes.cr

```diff
@@ -1238,6 +1261,11 @@ module Kubernetes
 
         field exec : Exec?
 
+        @[YAML::Field(key: "client-certificate-data")]
+        property client_certificate_data : String?
+        @[YAML::Field(key: "client-key-data")]
+        property client_key_data : String?
```

Comment:
The `field` macro takes a `key` argument to override the default camelCase version.

```suggestion
        field client_certificate_data : String?, key: "client-certificate-data"
        field client_key_data : String?, key: "client-key-data"
```

It's weird that Kubernetes uses both kebab-case and camelCase in property names, tbh, but I suppose my code isn't always consistent, either. 😄

Reply:
Oh, that's what I am looking. I came to Crystal doc about Serilisation and made use of property. I could not find the usage of `field`. Thanks for the hint.